scripts/st2bootstrap-deb.sh

```diff
@@ -497,7 +497,7 @@ quit();
 EOF
 
   # Require authentication to be able to acccess the database
-  sudo sh -c 'echo "security:\n  authorization: enabled" >> /etc/mongod.conf'
+  sudo sh -c 'printf "security:\n  authorization: enabled" >> /etc/mongod.conf'
```

Comment:
Looks like there is some small diff in results, when echo adds new line at the end, while printf doesnt:

```
vagrant@stackstorm:~$ sudo sh -c 'echo "security:\n  authorization: enabled"'
security:
  authorization: enabled
vagrant@stackstorm:~$ 
```

```
$ sudo sh -c 'printf "security:\n  authorization: enabled"'
security:
  authorization: enabledvagrant@stackstorm:~$ 
```

This could be a problem modifying `/etc/mongod.conf` this way. Let's please also add missing new line at the end to preserve the behavior.

Reply:
Good catch, I'll add that now